build.gradle

```diff
@@ -181,12 +191,13 @@ tasks.register('systemtest', Test) {
 }
 
 def serveTestReportInBackground = tasks.register('serveTestReportInBackground', com.github.psxpaul.task.JavaExecFork) {
-    dependsOn zipReport, compileJava, processResources, classes
+    dependsOn zipReport, compileJava, processResources, copyCypressConfig, copyMainClasses
     tasks.compileJava.mustRunAfter(zipReport)
     tasks.processResources.mustRunAfter(zipReport)
+    workingDir = 'build/serveTestReport'
     main = mainClassName
     classpath = sourceSets.main.runtimeClasspath
```

Comment:
I'm not 100% on this since my Gradle knowledge mainly comes from the past 10-20 minutes of googling, but is it possible that by using `sourceSets.main.runtimeClasspath` as a classpath, the copying of main classes is unnecessary? or rather i.e., does workingDir take precedence over the classpath stated?

Reply:
The reason why I copied the classes was to ensure that the necessary stuff for running the task was self-contained within the workingDir. Because I initially put `build/classes` and Gradle flagged it with a deprecation warning.